app/models/poll.rb

```diff
@@ -113,6 +113,18 @@ def undecided_count
 
   alias_method :user, :author
 
+  def parent_event
+    if discussion
+      discussion.created_event
+    else
+      nil
+    end
+  end
+
+  def created_event
+    events.find_by(kind: "poll_created")
```

Comment:
Symbol here too?

I wonder if there's some sort of future concern in here around objects which have creation events / sequence ids